portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,21 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  <body style="background-color:#94B4CB;">
+    <div id="header">
+      <h1>Hi, I'm Shreya.</h1>
     </div>
+    <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a onclick="showAbout()">About Me</a></li>
```

Comment:
I thought about that. I started them all off as links because I originally intended on creating multiple different pages, but then I realized I just wanted everything to pop up when the links were clicked, so I realized that I could have originally made them buttons. Since the last link is an actual link to my LinkedIn and since I started them all off as links, I thought I would keep them as links for consistency's sake. In my opinion, it's better to have them all as links than some as buttons, some as links; do you think the latter is a better approach?

Reply:
sounds good!